providers/pipedrive/parse.go

```diff
@@ -0,0 +1,34 @@
+package pipedrive
+
+import (
+	"strconv"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/jsonquery"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+	"github.com/spyzhov/ajson"
+)
+
+// nextRecordsURL builds the next-page url func.
+func nextRecordsURL(url *urlbuilder.URL) common.NextPageFunc {
```

Comment:
Can we add some link to documentation where it shows the data structure? Or example data so that we can see where `additional_data` `pagination` `more_items_in_collection` are coming from. This will also guide us when there is need for modification.